src/homepage/Homepage.tsx

```diff
@@ -1,71 +1,123 @@
-import { Container, createMuiTheme, makeStyles, Toolbar } from "@material-ui/core"
+import {
+    Container,
+    createMuiTheme,
+    makeStyles,
+    Toolbar,
+} from "@material-ui/core"
 import { useEffect, useState } from "react"
 import PointsCard from "../leaderboard/PointCard"
 import League from "../league/League"
 import Header from "../misc/Header"
 import Predictions from "../predictions/Predictions"
-import { IMatchData } from "../types/types"
+import { ILeague, IMatchData } from "../types/types"
 import { getJWT, resolveEndpoint } from "../utils/Utils"
 import React from "react"
 
 const useStyles = makeStyles({
     logo: {
-        maxHeight: "15vw"
+        maxHeight: "15vw",
     },
     homepage: {
         position: "relative",
         left: 0,
-        width: "100%"
-    }
+        width: "100%",
+    },
 })
 
 export const fontTheme = createMuiTheme({
     typography: {
-        fontFamily: [
-            "Source Sans Pro",
-            "sans-serif",
-        ].join(","),
+        fontFamily: ["Source Sans Pro", "sans-serif"].join(","),
```

Comment:
this seems silly

Reply:
aww fuck knows what it's doing

Reply:
i think it's probably cleaner if you have loads of fonts or smthn